Core/Utilities/Graphs/Legend.py

```diff
@@ -22,7 +22,9 @@
 class Legend:
 
   def __init__(self,data=None,axes=None,*aw,**kw): 
-
+    
+    self.text_size = 0
+    self.column_width = 0
```

Comment:
I just declared the variable, this was used as it is now. The problem is if we want to change the variable name, we have to completely review the plotting package. All variables are declared this way. Even the default values which in the __init__.py.